core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/heuristic/selector/move/generic/list/kopt/KOptListMoveSelectorFactory.java

```diff
@@ -35,23 +31,20 @@ protected MoveSelector<Solution_> buildBaseMoveSelector(HeuristicConfigPolicy<So
             SelectionCacheType minimumCacheType, boolean randomSelection) {
         SelectionOrder selectionOrder = SelectionOrder.fromRandomSelectionBoolean(randomSelection);
         EntitySelectorConfig entitySelectorConfig = new EntitySelectorConfig();
-        // TODO let's not forget this here
-        //entitySelectorConfig.setCacheType(SelectionCacheType.PHASE);
+        entitySelectorConfig.setCacheType(SelectionCacheType.PHASE);
```

Comment:
Are we sure about this?
Phase cache is a tall order, can you prove that the cache stays fresh throughout?

Reply:
I can say it does not crash + generate new moves, at the very least. Unsure about freshness. It was a copied TODO/commented out code from another MoveSelectorFactory I copied. I can remove it.

Reply:
Please do. Unless we can prove it is safe, let's not use it.